### PR TITLE
Change in the help Keptn upgrade command for the new format of shipyard. #3479

### PR DESCRIPTION
**Pull request by adigeak**
Signed-off-by: adigeak <adigeak29@gmail.com>

### CODE REVIEW

File: cli/cmd/upgrade.go [hunk 1]
```diff
@@ -49,8 +49,9 @@ var upgraderCmd = NewUpgraderCommand(version.NewKeptnVersionChecker())
 func NewUpgraderCommand(vChecker *version.KeptnVersionChecker) *cobra.Command {
 	upgradeCmd := &cobra.Command{
 		Use:   "upgrade",
-		Short: "Upgrades Keptn on a Kubernetes cluster",
-		Long: `The Keptn CLI allows upgrading Keptn on any Kubernetes derivative to which your kube config is pointing to, and on OpenShift.
+		Short: "Upgrades Keptn on a Kubernetes cluster and also the project to new format.",
+		Long: `The Keptn CLI allows upgrading Keptn on any Kubernetes derivative to which your kube config is pointing to, and on OpenShift. 
+		Also upgrade each of your existing projects to the new format for the shipyard.
```
Comment (johannes-b) [inline cli/cmd/upgrade.go:54]: Please change to: `The Keptn CLI allows upgrading Keptn on any Kubernetes derivative to which your kube config is pointing to, and on OpenShift. Also, it supports upgrading the shipyard of an existing project to a new specification.`

File: cli/cmd/upgrade.go [hunk 2]
```diff
@@ -49,8 +49,9 @@ var upgraderCmd = NewUpgraderCommand(version.NewKeptnVersionChecker())
 func NewUpgraderCommand(vChecker *version.KeptnVersionChecker) *cobra.Command {
 	upgradeCmd := &cobra.Command{
 		Use:   "upgrade",
-		Short: "Upgrades Keptn on a Kubernetes cluster",
-		Long: `The Keptn CLI allows upgrading Keptn on any Kubernetes derivative to which your kube config is pointing to, and on OpenShift.
+		Short: "Upgrades Keptn on a Kubernetes cluster and also the project to new format.",
```
[the remark this answers is not in the document]
Comment (adigeak) [reply cli/cmd/upgrade.go]: done. my thinking was to make is short.

**Comment by johannes-b** [comment]
@adigeak thanks for incorporating the changes. Can you please sign your last commit as explained here: https://github.com/keptn/keptn/runs/2717579237

**Comment by adigeak** [comment]
Done.  I thought we have to do this one's.

**Comment by johannes-b** [comment]
@adigeak unfortunately, it did not work. and the last commits are not signed. You can see this by the red X next to the comment: 
image

Can you please try it again.

**Comment by adigeak** [comment]
I think it's done.

**Comment by johannes-b** [comment]
Unfortunately not. I'm sorry for the inconvenience, but the last commits are unsigned. 

I really would like to help you getting this PR merged and too avoid wasting too much time on the DCO, may I ask you to: 
* Create a new branch, e.g.: `adigeak:feature/3479/ImproveHelpMessage-2`
* Add the change to the file `cli/cmd/upgrade.go`
* Do a `git commit -s -m "your message"`  (Please note the `-s` !) 
* Open a new PR.

**Comment by adigeak** [comment]
closing this pr. New pr created #4231

**Pull request closed without merging**